source/jormungandr/jormungandr/street_network/taxi.py

```diff
@@ -118,7 +118,7 @@ def _add_additional_section_in_fallback(
                     journey, pt_object_destination, request["additional_time_after_first_section_taxi"]
                 )
             else:
-                self._add_additional_section_before_first_section_taxi(
+                self._add_additional_section_before_last_section_taxi(
```

Comment:
shouldn't it be
```suggestion
                self._add_additional_section_after_last_section_taxi(
```
?

Reply:
Nope, want you use taxi as last_section, there is an additional waiting section between pt and taxi.